### PR TITLE
Create success modal

### PR DESCRIPTION
**Pull request by IanMuler**
# What is the objective?

Reduce the amount of cost necessary to start using the digital menu. Improve the onboarding process and automatically create the digital menu with great quality content.

# User story

As an new registered user, I want be guided once I get the menu page generated. So I can start customizing my QR menu.

# Acceptance criteria

- ~User can see the modal on the images page.~
- ~Modal is shown once the menu is generated.~
- ~Links should work fine.~

# Technical Notes

None

# What should be tested

- ~Links should navigate to the admin pages like products and QR setting~
- Modal animation

# QA Notes

Explain how to test newly implemented features and what are steps required.

1. Visit the shop review page https://url.com/
2. Check if reviews get rendered
3. Check sort and filter buttons

### CODE REVIEW

File: cypress.config.ts [hunk 1]
```diff
@@ -12,5 +12,6 @@ export default defineConfig({
         "https://motion-menu-api-staging.herokuapp.com",
     },
     supportFile: false,
+    chromeWebSecurity: false, //to allow cross origin requests
```
Comment (briankangdev) [inline cypress.config.ts:15]: Por qué es necesario? @IanMuler

File: cypress/e2e/Images_page.cy.js [hunk 2]
```diff
@@ -1,70 +1,122 @@
...
+    //   it("render 'all reviews' button", () => {
+    //     cy.get("[data-testid=all-reviews]").should("exist");
+    //   });
+    // });
+
+    // context("menu", () => {
+    //   it("render menu masonry", () => {
+    //     cy.get("[data-testid=menu]").should("exist");
+    //   });
+    // });
```
Comment (briankangdev) [inline cypress/e2e/Images_page.cy.js]: Tendrías que sacar los comentarios o arregralos.

File: src/routes/shop/[company_id]/images/+page.svelte [hunk 3]
```diff
@@ -68,6 +70,13 @@
 
       menu_ref.style.height = `${max_height - products_height}px`;
     }
+
+    //show success notification if the user is coming from the loading page
+    if (history.state.from === "loading") {
+      toast(SuccessNotification as any, {
+        duration: 5000,
+      });
+    }
```
Comment (briankangdev) [inline src/routes/shop/[company_id]/images/+page.svelte:61]: 👍

Comment (briankangdev) [reply src/routes/shop/[company_id]/images/+page.svelte:61]: No hay otra manera más clara? porque loading es muy general.
cookies localstorage o url params.

File: cypress.config.ts (hunk 1 above)
Comment (IanMuler) [reply cypress.config.ts:15]: Para que permita viajar a link como https://admin.motion.menu que no son parte del localhost

File: cypress/e2e/Images_page.cy.js (hunk 2 above)
Comment (IanMuler) [reply cypress/e2e/Images_page.cy.js]: Sí, no me di cuenta. Ahí agrego el commit

File: src/routes/shop/[company_id]/images/+page.svelte (hunk 3 above)
Comment (IanMuler) [reply src/routes/shop/[company_id]/images/+page.svelte:61]: Ahora agrego url params, estuve viendo que es más común de ese modo

Comment (briankangdev) [reply src/routes/shop/[company_id]/images/+page.svelte:61]: `const shop_is_new = url.searchParams.get("new");`
Cambiamos el nombre no más, parece está perfecto.
`const shop_is_new = url.searchParams.get("new_shop_owner");`

**Review by briankangdev — APPROVED** [review]

**Pull request merged**